### PR TITLE
Optimized two large images.

### PR DESCRIPTION
**Pull request by Yeen125**
I have optimized the "organizations" and "volunteers" images by resizing them and converting them to JPG from PNG.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,17 +59,30 @@ <h1 class="text-center" id="workWithUs">Work with us</h1>
           <div class="card-info">
             <h4 class="cardTitle">ORGANIZATIONS</h4>
             <h2 class="cardText">Are you a Chicagoland nonprofit that needs some tech help?</h2>
+<<<<<<< Updated upstream
```
Comment (jrapala) [inline index.html:62]: Please remove these stash markers

**Review by jrapala — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Yeen125) [reply index.html:62]: I think these slash markers were created by GitHub Desktop.

Comment (Yeen125) [reply index.html:62]: If there is anything else I am doing wrong, let me know.

Comment (jrapala) [reply index.html:62]: Can you please delete the markers from the file?

**Pull request merged**